ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -128,79 +159,68 @@ - (void)updateConstraints {
-                                                                      relatedBy:NSLayoutRelationEqual
-                                                                         toItem:self.slider
-                                                                      attribute:NSLayoutAttributeCenterX
-                                                                     multiplier:1.0
-                                                                       constant:-kBigMargin];
-        [self addConstraint:constraint];
-        constraint = [NSLayoutConstraint constraintWithItem:self.leftRangeLabel
-                                                  attribute:NSLayoutAttributeRight
-                                                  relatedBy:NSLayoutRelationEqual
-                                                     toItem:self.slider
-                                                  attribute:NSLayoutAttributeCenterX
-                                                 multiplier:1.0
-                                                   constant:-kBigMargin];
-        [self addConstraint:constraint];
-    }
-    else
-    {
-        // Horizontal slider constraints
-        const CGFloat kMargin = 17.0;
-        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[_valueLabel(==40)]-[_slider]"
-                                                                     options:NSLayoutFormatAlignAllCenterX
-                                                                     metrics:nil
-                                                                       views:views]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:self.leftRangeLabel
+                                                         attribute:NSLayoutAttributeRight
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self.slider
+                                                         attribute:NSLayoutAttributeCenterX
+                                                        multiplier:1.0
+                                                          constant:-kSideLabelMargin]];
         
-        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-kMargin-[_leftRangeLabel]-kMargin-[_slider]-kMargin-[_rightRangeLabel(==_leftRangeLabel)]-kMargin-|"
-                                                                     options:NSLayoutFormatAlignAllCenterY|NSLayoutFormatDirectionLeadingToTrailing
-                                                                     metrics:@{@"kMargin": @(kMargin)}
-                                                                       views:views]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:self.rightRangeLabel
+                                                        attribute:NSLayoutAttributeCenterY
```

Comment:
This is a slightly modified version of what was in #33.  The labels for the min and max in vertical mode get centered vertically relative to the top and bottom points of the slider line, thus the use of CenterY for the labels and Top and Bottom for the slider.